doc/input.md

```diff
@@ -102,13 +102,13 @@ class MyGame extends Game with TapDetector {
   // Other methods omitted
 
   @override
-  void onTapDown(TapDownDetails details) {
-    print("Player tap down on ${details.globalPosition.dx} - ${details.globalPosition.dy}");
+  void onTapDown(TapDownInfo event) {
+    print("Player tap down on ${event.position.x} - ${event.position.y}");
```

Comment:
nit: we can just print `event.position`, I imagine Vector2 has a nice toString (same below)